enterprise/frontend/src/embedding-sdk/components/public/StaticDashboard/StaticDashboard.tsx

```diff
@@ -0,0 +1,85 @@
+import { pick } from "underscore";
+
+import {
+  DEFAULT_EMBED_DISPLAY_OPTIONS,
+  useDashboardFullscreen,
+  useDashboardRefreshPeriod,
+  useEmbedTheme,
+  useRefreshDashboard,
+} from "metabase/dashboard/hooks";
+import type {
+  EmbedDisplayParams,
+  RefreshPeriod,
+} from "metabase/dashboard/types";
+import { isNotNull } from "metabase/lib/types";
+import { PublicDashboard } from "metabase/public/containers/PublicDashboard/PublicDashboard";
+import { Box } from "metabase/ui";
+import type { DashboardId, ParameterId } from "metabase-types/api";
+
+export const StaticDashboard = ({
+  dashboardId,
+  parameterValues,
+  refreshPeriod: initialRefreshPeriod = null,
+  bordered,
+  titled,
+  theme: userTheme,
+  font,
+  hideDownloadButton,
+  hideParameters,
+}: {
+  dashboardId: DashboardId;
+  parameterValues: Record<ParameterId, string | string[] | null | undefined>;
+  refreshPeriod?: RefreshPeriod;
+} & Partial<EmbedDisplayParams>) => {
+  const options: EmbedDisplayParams = {
+    ...DEFAULT_EMBED_DISPLAY_OPTIONS,
+    ...pick(
+      {
+        bordered,
+        titled,
+        theme: userTheme,
+        font,
+        hideDownloadButton,
+        hideParameters,
+      },
+      isNotNull,
+    ),
+  };
+
+  const { refreshDashboard } = useRefreshDashboard({
+    dashboardId,
+    queryParams: parameterValues,
```

Comment:
just a naming thing. Should we call the parameter `parameterValues` instead of `queryParams`? `queryParams` sounds more generic and there's also a notion of URL query params as well which might make this a bit confusing?

Reply:
I buy that name 💪